source/getting_started/step-by-step.rst

```diff
@@ -1,10 +1,55 @@
 .. _step-by-step:
 
+
+.. |checkbox| raw:: html
+
+    <input type="checkbox">
+
+
 ============
 Step by step
 ============
 
 
+Checklist
+=========
+
+Setting up ETOS is a daunting task when you are doing it the first time around.
+But it is also easy to miss a few steps that are required for it to work.
+
+For this reason we have created a checklist in order to keep track of what needs to be done so that no steps are forgotten.
+
+* |checkbox| Set up a Kubernetes cluster (ETOS runs here)
+* |checkbox| Set up an execution space (Execute test runner containers)
+* |checkbox| Set up a log area (Storing logs after execution)
+* |checkbox| Deploy MongoDB (Event storage DB)
+* |checkbox| Deploy Redis Sentinel (ETOS internal communications)
+* |checkbox| Deploy RabbitMQ (must be accessible outside of kubernetes as well)
+* |checkbox| Deploy `Eiffel GraphQL API <https://eiffel-graphql-api.readthedocs.io/en/latest/examples.html#start-api-docker>`_
+
+  * |checkbox| Deploy an Eiffel event storage tool. `Example <https://eiffel-graphql-api.readthedocs.io/en/latest/examples.html#start-storage-docker>`_
+
+* |checkbox| Deploy ETOS helm chart: :ref:`index:Installation`
+* |checkbox| Configure :ref:`etos-environment-provider`
+
+   * |checkbox| :ref:`services/etos_environment_provider:IUT Provider`
+
+   * |checkbox| :ref:`services/etos_environment_provider:Execution Space Provider`
+
+   * |checkbox| :ref:`services/etos_environment_provider:Log Area Provider`
+
+* |checkbox| Create a :ref:`tercc`
```

Comment:
We should call this ETOS Test Collection instead. TERCC is from the eiffel event name but I believe we never really tell that story in the documentation.

Reply:
Done. Link is still named 'tercc' so that no other parts break.